requirements.txt

```diff
@@ -6,3 +6,6 @@ Pillow>=7.0,<9.6
 pulpcore>=3.23.0,<3.25
 PyYAML>=5.4.1,<7.0
 semantic_version>=2.9,<2.11
+setuptools>=39.2,<66.2.0
+sigstore==1.1.1
+voluptuous==0.13.1
```

Comment:
Is `voluptuous` still being used? Also what is `setuptools` being used for? I would like to keep that out the requirements.